modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/transaction/Filters.kt

```diff
@@ -91,8 +111,15 @@ object Filters {
     }
 }
 
+/**
+ * `EntityFilters` match events associated with a given entity
+ * and either return all matching events or additionally apply another filter to them.
+ */
 object EntityFilters {
 
+    /**
+     * Match events associated with asset definition and apply [idFilter] or [eventFilter]
```

Comment:
I'd rephrase artificial parameter names to be included as human readable stuff in the docs, e.g. filter referenced by its [id][IdFilterAssetDefinitionId]

Reply:
Updated